pyartcd/pyartcd/constants.py

```diff
@@ -17,5 +17,11 @@
     "ppc64le": "registry.ci.openshift.org/ocp-ppc64le/release-ppc64le",
     "aarch64": "registry.ci.openshift.org/ocp-arm64/release-arm64",
 }
+MIRROR_CLIENTS = {
```

Comment:
Can't tell what this is used for. Would want a more descriptive name or comment above it

Reply:
add comments